src/components/LoadImage.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useRef } from "react";
+import PropTypes from "prop-types";
+
+const LoadImage = ({ src, alt, className }) => {
```

Comment:
Can we get a tutorial pls 😄 because we don't know how it works exactly.
// Martin, Linda

Reply:
Me too 😉